.gitignore

```diff
@@ -13,3 +13,7 @@ demo/bundle.js
 coverage
 tests/integration/__pdfs__
 tests/integration/pdfmake/__pdfs__
+tests/integration/**/__snapshots__/*
+tests/integration/__snapshots__/*
+*.snap
+/*.pdf
```

Comment:
No problem. I think the snapshots are not even being used. It would be best if we cleaned up the integration or unit tests that are causing these to be generated. Alas this is my first _jest_ project (have used mocha in the past) so I'd prefer not to make cleanup mistakes.

Reply:
Reverted